frontend/src/javascript/utils/googleLoginUtils.js

```diff
@@ -0,0 +1,34 @@
+import { sendRequest } from "@/javascript/utils/axiosUtils";
```

Comment:
Na frontendzie unikałbym jakiegoś podziału funkcjonalności ze względu na providera. Zwłaszcza funkcjonalności takie jak sprawdzenie czy użytkownik jest zalogowany oraz wylogowanie użytkownika powinno być niezależne od czy jesteśmy zalogowani za pomocą googla czy jakiegoś innego providera (chodzi mi tutaj głównie o nazewnictwo funkcji oraz endpointy.

Reply:
No to mogę poprawić dla logout i verify ale login będzie inne dla różnych usług (nawet z Passportem) więc przynajmniej logowanie musi mieć podział.

Reply:
Jasne logowanie może mieć podział, ale logout i verify to powinno być wspólne.